packages/core/src/modal/modal-actions.element.ts

```diff
@@ -38,11 +38,9 @@ export class CdsModalActions extends LitElement {
   }
 
   render() {
-    return html`
-      <div cds-layout="${this.hasAttribute('cds-layout') ? 'display:contents' : 'horizontal gap:sm align:right'}">
-        <slot></slot>
-      </div>
-    `;
+    return this.hasAttribute('cds-layout')
+      ? html`<slot></slot>`
+      : html`<div cds-layout="horizontal gap:sm align:right"><slot></slot></div>`;
```

Comment:
If there is a cds-layout on host use the default slot with `display: contents`. This fixes the issue in IE since the slot tag is removed and the content will then become the direct child of the host element.